src/modes/install.ts

```diff
@@ -39,15 +46,52 @@ export default async function(pkgs: PackageRequirement[]) {
         if (program.includes("{{")) continue
 
         const pkgstr = utils.pkg.str(pkg)
-        const exec = `exec pkgx +${pkgstr} -- ${program} "$@"`
-        const script = undent`
-          if [ "$PKGX_UNINSTALL" != 1 ]; then
-            ${exec}
-          else
-            cd "$(dirname "$0")"
-            rm ${programs.join(' ')} && echo "uninstalled: ${pkgstr}" >&2
-          fi`
-        const f = dst.mkdir('p').join(program)
+
+        let script = ""
+
+        if (UNSAFE) {
+          const config = hooks.useConfig()
+          const parts = pkgstr.split("/")
+          parts.pop()
```

Comment:
this is probably better done with slice, but that's just a style choice

Reply:
Done